sdks/docker-enclave/default.nix

```diff
@@ -10,6 +10,8 @@
   vet,
   kernels,
   compose ? ./. + "/docker-compose.yml",
+  dockerImageTar ? ./. + "/docker-image.tar",
```

Comment:
should be an array to pass in multiple tars

Reply:
done